packages/leva/src/components/UI/Label.tsx

```diff
@@ -32,17 +32,19 @@ function RawLabel(props: LabelProps) {
     <>
       {optional && <OptionalToggle />}
       {hint !== undefined ? (
-        <RadixTooltip.Root>
-          <RadixTooltip.Trigger asChild>
-            <StyledLabel {...htmlFor} {...props} />
-          </RadixTooltip.Trigger>
-          <RadixTooltip.Content side="top" sideOffset={2}>
-            <StyledToolTipContent>
-              {hint}
-              <ToolTipArrow />
-            </StyledToolTipContent>
-          </RadixTooltip.Content>
-        </RadixTooltip.Root>
+        <RadixTooltip.Provider>
```

Comment:
a `RadixTooltip.Provider` was introduced https://www.radix-ui.com/docs/primitives/components/tooltip#provider

There may be a better place to place it (to share global tooltips behavior like `delayDuration`, cf. doc)

I let you decide for this ;)